ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/tools/osb/mbean/ObjStoreBrowser.java

```diff
@@ -315,8 +369,8 @@ public UidWrapper findUid(Uid uid) {
     }
 
     /**
-     * Find the registered beand corresponding to a uid.
-     * @deprecated use {@link #findUid(com.arjuna.ats.arjuna.common.Uid)} ()} instead.
+     * Find the registered beand corresponding to an uid.
```

Comment:
`a uid` is the correct usage of the indefinite article since it precedes a consonant.

Reply:
I see, the spellchecker told me opposite. Adjusted it back, thanks!

Reply:
The idea spell checker says the same but it it is incorrect, `a uid` is the correct indefinite article to use.